testdata/project-v2-addon/config/default/kustomization.yaml

```diff
@@ -13,15 +13,16 @@ namePrefix: project-v2-addon-
 #  someName: someValue
 
 bases:
+- ../manager
 - ../crd
 - ../rbac
-- ../manager
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in 
+# +kubebuilder:scaffold:bases
+# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
```

Comment:
We cannot change V2/v2.0.0 project layouts. 
With the plugins introduction, we need to address these changes just for V3-alpha projects layouts with new bump plugins versions. See: https://github.com/kubernetes-sigs/kubebuilder/pull/1498